storage/config.go

```diff
@@ -0,0 +1,14 @@
+package storage
```

Comment:
I see that you are using constants as well (clients/pastel/config.go). that is why I thought that if the client/user missed this config or deleted it, then we set the default value.

Reply:
This is right, but I didn't understand that is default value, you should have called it for example `defaultDBPath`
And since the field doesn't have `tag`, that means it will never be loaded from file

Reply:
I have already fixed the bug related with tag